tests/test_wire/test_undirected_error.py

```diff
@@ -0,0 +1,42 @@
+import magma as m
+from magma.testing.utils import has_warning
+
+
+def test_undirected_value_warning_1(caplog):
+    m.config.set_debug_mode(True)
```

Comment:
we might want to use `with magma_debug_section(): ...` here (from `magma.testing.utils`) so as to not keep debug mode on for other tests unexpectedly. Not sure if we already turn it off before each test elsewhere (e.g. conftest).